Cargo.toml

```diff
@@ -23,6 +23,9 @@ unstable = []
 # musl libc.
 musl-reference-tests = ['rand']
 
+# Used to prevent using any intrinsics or arch-specific code.
+only-soft-floats = []
```

Comment:
Can you rename this to `force-soft-float`.

Reply:
Done!